src/demos/shell/demo-app-advanced-2.html

```diff
@@ -131,7 +131,7 @@
               </calcite-action-group>
             </calcite-action-bar>
               <calcite-panel
-                width-scale="m"
+                
```

Comment:
Nitpick: can you remove this empty newline?

Reply:
Will do.